documentation/modules/rn-2.2.adoc

```diff
@@ -9,10 +9,13 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
+[id="technical-changes-22_{context}"]
+== Technical changes
 //
-// .TBD
+.Setting interval between precopies
+
+The precopy interval is now set as an environment variable on the forklift-controller pod's `controller` container. In order to adjust it, you need to add `PRECOPY_INTERVAL` with a value in minutes to the `env` section of the `controller` container definition on the `forklift-controller` deployment, or add the key `PRECOPY_INTERVAL` with a value in minutes to the `forklift-controller-config` config map and restart the controller pod.
```

Comment:
You do not need to document the feature. You just need to say that the feature is supported.